File: 1264-maximum-number-of-words-you-can-type/1264-maximum-number-of-words-you-can-type.java
class Solution {
    public int canBeTypedWords(String text, String brokenLetters) {
        HashMap<Character, Boolean> hmap = new HashMap<>();

        for(char ch : brokenLetters.toCharArray()) {
            hmap.put(ch, true);
        }
        String words[] = text.split(" ");
        int count = 0;
        for(String word : words) {
            boolean canType = true;
            for(char ch : word.toCharArray()) {
                if(hmap.containsKey(ch)) {
                    canType = false;
                    break;
                }
            }

            if(canType) count++;
        }
        return count;
    }

}

File: 1264-maximum-number-of-words-you-can-type/1264-maximum-number-of-words-you-can-type.py
class Solution:
    def canBeTypedWords(self, text: str, brokenLetters: str) -> int:
        hmap = {}

        for ch in brokenLetters:
            hmap[ch] = True

        words = text.split(" ")
        count = 0
        
        for word in words:
            canType = True
            for ch in word:
                if ch in hmap:
                    canType = False
                    break
            
            if canType:
                count = count + 1

        return count

        